### PR TITLE
CDK setup for the application

### PR DESCRIPTION
**Pull request by Vivek205**

### CODE REVIEW

File: cdk/lib/airdrop-stack.ts [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+export class AirdropStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    builder
+      .build()
+      .then(() => {
+        // Lambda functions for handling edge page requests
+        const defaultLambda = new lambda.Function(this, defaultLambdaName, {
+          runtime: lambda.Runtime.NODEJS_12_X,
```
Comment (rajeevbbqq) [inline cdk/lib/airdrop-stack.ts:50]: can't we use NODE 14.x ?

File: cdk/lib/airdrop-stack.ts [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+console.log("build options", options);
+
+const builder = new Builder(nextConfigDir, outputDir, options);
+
+const defaultLambdaName = "defaultEdgeLambda";
+const apiLambdaName = "apiLambdaName";
+const imageLambdaName = "imageEdgeLambda";
+const bucketName = "bucketName";
+const distributionName = "myDist";
+const imageCachePolicyName = "imageCachePolicy";
```
Comment (rajeevbbqq) [inline cdk/lib/airdrop-stack.ts:39]: can we keep these config in `.env` or in a separate config file ?

**Review by rajeevbbqq — COMMENTED** [review]
Added few comments. 
Please delete `yarn.lock` if not required.

**Comment by Vivek205** [comment]
> Added few comments.
> Please delete `yarn.lock` if not required.

It is better to keep yarn.lock as it improves in caching the packages in CI/CD

**Review by rajeevbbqq — APPROVED** [review]

**Pull request merged**